package.json

```diff
@@ -7,7 +7,6 @@
     "audit": "npm audit --audit-level=moderate",
     "audit-licenses": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
     "build": "next build && next export",
-    "ci": "npm run lint && npm run test",
```

Comment:
I'd never recommend naming a command in a way that conflicts with a built-in npm command — in this case `npm ci` exists, so I wouldn't suggest having a `npm run ci` command; By splitting these out, we also have clearer test output.

Reply:
If nobody's using this, it's fine (`npm run ci` was a habit I'd gotten into before `npm ci` existed.) We should have a discussion about having a simple command to run audit/lint/test in localhost, but as husky runs lint and test it's not a problem for now. We've discussed removing husky elsewhere so that probably isn't the right answer long term.